### PR TITLE
Genre class and Unit Test

### LINKED ISSUE #9: [2pt][Student2]: Add unit tests
- Add unit tests for all implemented methods.

### PR DESCRIPTION
**Pull request by Yorsyboy**
### In this PR, I

-  Created Genre class.
-  Added the association relationship between Genreand Item.
-  Added the method for "list all Genre" in main.rb.
-  Added methods for saving and loading genresto/from a .json file.
-  Added unit tests

Closes #9 #8

### CODE REVIEW

File: music_album.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def add_music_album
     end
 
     music_albums.push({ 'id' => @id, 'on_spotify' => @on_spotify, 'publish_date' => @publish_date,
-                        'archived' => @archived })
+                        'author' => @author, 'Music_genre' => @genre, 'archived' => @archived })
```
Comment (ha-manel) [inline music_album.rb]: Since you added `author` and `genre` to the file, kindly add `label` too to the hash 😊

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 
### Hey @Yorsyboy 
Great work! 👏🏻  the tests have good coverage and the app is working as intended ✔️  However, there is a tiny change to implement before we can merge this.

File: music_album.rb (hunk 1 above)
Comment (Yorsyboy) [reply music_album.rb]: Thanks for the review, Will work on it

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅ 
### Congratulations 🎉 🎉 
It's time to merge!

**Pull request merged**